IGInterfaceDataTable/WKInterfaceTable+IGInterfaceDataTable.m

```diff
@@ -341,6 +381,39 @@ - (void)insertRowsAtIndexPaths:(NSArray *)indexPaths withRowType:(NSString *)row
   }
 }
 
+- (void)beginUpdates {
+  self.updating = true;
+  self.rowIndexesToAdd = [[NSMutableArray alloc] init];
+  self.rowTypesToAdd = [[NSMutableDictionary alloc] init];
+}
+
+- (void)endUpdates {
+  if(!self.updating) {
+    return;
```

Comment:
Let's also add an assert here above the if-statement so that if you're debugging you can catch that begin/end updates aren't being called in sync:
```
NSAssert(self.updating, @"Calling -endUpdates without first calling -beginUpdates.");
```